src/globus_cli/commands/flows/start.py

```diff
@@ -0,0 +1,141 @@
+
+import typing as t
+import uuid
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import JSONStringOrFile, command, flow_id_arg
+from globus_cli.termio import Field, TextMode, display, formatters
+
+ROLE_TYPES = ("flow_viewer", "flow_starter", "flow_administrator", "flow_owner")
+
+
+@command("start", short_help="Start a flow")
+@flow_id_arg
+@click.option(
+    "--input",
+    "input_document",
+    type=JSONStringOrFile(),
+    help="""
+        The JSON input parameters used to start the flow.
+
+        The input document may be specified inline,
+        or it may be a path to a JSON file, prefixed with "file:".
+
+        Example: Inline JSON:
+
+        \b
+            --input '{"src": "~/source"}'
+
+        Example: Path to JSON file:
+
+        \b
+            --input file:parameters.json
+
+        If unspecified, the default is an empty JSON object ('{}').
+    """,
+)
+@click.option(
+    "--label",
+    type=str,
+    help="A label to give the run.",
+)
+@click.option(
+    "--manager",
+    "managers",
+    type=str,
+    multiple=True,
+    help="""
+        A principal that may manage the execution of the run.
+
+        This option can be specified multiple times
+        to create a list of run managers.
+    """,
+)
+@click.option(
+    "--monitor",
+    "monitors",
+    type=str,
+    multiple=True,
+    help="""
+        A principal that may monitor the execution of the run.
+
+        This option can be specified multiple time
+        to create a list of run monitors.
+    """,
+)
+@click.option(
+    "--tag",
+    "tags",
+    type=str,
+    multiple=True,
+    help="""
+        A tag to associate with the run.
+
+        This option can be used multiple times
+        to create a list of tags.
+    """,
+)
+@LoginManager.requires_login(LoginManager.FLOWS_RS)
+def start_command(
+    login_manager: LoginManager,
+    flow_id: uuid.UUID,
+    input_document: dict | None | t.Any,
+    label: str,
+    managers: tuple[str],
+    monitors: tuple[str],
+    tags: tuple[str],
+):
+    """
+    Start a flow.
+
+    FLOW_ID must be a UUID.
+    """
+
+    if input_document is None:
+        input_document = {}
+
+    flow_client = login_manager.get_specific_flow_client(flow_id)
+    response = flow_client.run_flow(
+        body=input_document,
+        label=label,
+        tags=list(tags),
+        run_managers=list(managers),
+        run_monitors=list(monitors),
+    )
+
+    auth_client = login_manager.get_auth_client()
+    principal_formatter = formatters.auth.PrincipalURNFormatter(auth_client)
+
+    fields = [
+        Field("Flow ID", "flow_id"),
+        Field("Flow title", "flow_title"),
+        Field("Run ID", "run_id"),
+        Field("Label", "label"),
+        Field(
+            "Run owner",
+            "run_owner",
+            formatter=formatters.auth.PrincipalURNFormatter(
+                login_manager.get_auth_client()
+            ),
```

Comment:
A somewhat unnecessary nitpick, but I believe you can...
```suggestion
            formatter=principal_formatter
```

Reply:
Ah, got it. I saw this pattern in a nearby file and followed it, but I see the pattern in `show.py` now (and your comment below) and I'm following that, now.